chsdi/templates/htmlpopup/cadastralwebmap.mako

```diff
@@ -22,12 +22,14 @@
   params = CadastralWebMapParams(request)
 
   c['bbox'] = params.mapExtent.bounds
-  c['bboxlv95'] = [c['bbox'][0] + 2000000, c['bbox'][1] + 1000000,
-                   c['bbox'][2] + 2000000, c['bbox'][3] + 1000000]
+  c['bboxlv03'] = [c['bbox'][0] - 2000000, c['bbox'][1] - 1000000,
+                   c['bbox'][2] - 2000000, c['bbox'][3] - 1000000]
   c['scale']  = get_scale(params.imageDisplay, params.mapExtent)
   defaultCoord = [(c['bbox'][0] + c['bbox'][2]) / 2,
                   (c['bbox'][1] + c['bbox'][3]) / 2]
   clickCoord = request.params.get('coord').split(',') if request.params.get('coord') else defaultCoord
+  c['clickCoordLv03'] = [clickCoord[0][1:],
+                         clickCoord[1][1:]]
```

Comment:
why do you use [1:] ? coords in this case is also a list of 2 coordinates which srid is determined by `params.srid`
so make sure you clearly have both `clickCoordLv03` and `clickCoordLv95` in the Mako context because `clickCoord` could be either LV95 or LV03.

Reply:
clickCoord returns a tuple of strings which represents the coordinates in LV95. As I needed the coordinates in LV03 I removed the 2's (east) and the 1's (north) by shortening the string by its first character. I agree, it's not a very nice hack :D 
Anyway, I will in a next step add it to the context variable such that it is not necessary anymore to apply that .